{"name": "test_query_by_status_No", "status": "failed", "statusDetails": {"message": "AssertionError: assert '禁用' in []\n +  where [] = <bound method Sys_user_manage.get_User_table_value_by_key of <Page.sys_user_manage.Sys_user_manage object at 0x000001FDD1F8A2E8>>(key='状态', index=1)\n +    where <bound method Sys_user_manage.get_User_table_value_by_key of <Page.sys_user_manage.Sys_user_manage object at 0x000001FDD1F8A2E8>> = <Page.sys_user_manage.Sys_user_manage object at 0x000001FDD1F8A2E8>.get_User_table_value_by_key", "trace": "self = <Testsuit.System_Settings.User_Manage.Test_User_add_update_query.Test_user_add_update_query object at 0x000001FDD1F505F8>\nSys_user_manage_page = <Page.sys_user_manage.Sys_user_manage object at 0x000001FDD1F8A2E8>\n\n    @pytest.mark.run(order=24)\n    @pytest.mark.skipif(is_skip == 0,reason = 'test')\n    @allure.story('通过状态查询已禁用角色')\n    def test_query_by_status_No(self,Sys_user_manage_page):\n        '''\n                  测试用例编号:\n                  测试用例标题:通过状态查询已启用角色\n                  前置条件：打开编辑账号信息界面\n                  步骤：\n                  \"\n                  1.选择状态“已禁用”\n                  2.点击查询按钮\n    \n                  预期：\n                  1. 可以选择\n                  2. 返回模糊查询结果\n    \n        '''\n    \n        #刷新界面\n        Sys_user_manage_page.refresh()\n    \n        #选择状态“已禁用”\n        Sys_user_manage_page.select_status_by_index(index = 2)\n    \n        #点击查询按钮\n        Sys_user_manage_page.query_click()\n    \n        #断言\"禁用\"是否在列表中\n>       assert '禁用' in Sys_user_manage_page.get_User_table_value_by_key(key = '状态', index = 1)\nE       AssertionError: assert '禁用' in []\nE        +  where [] = <bound method Sys_user_manage.get_User_table_value_by_key of <Page.sys_user_manage.Sys_user_manage object at 0x000001FDD1F8A2E8>>(key='状态', index=1)\nE        +    where <bound method Sys_user_manage.get_User_table_value_by_key of <Page.sys_user_manage.Sys_user_manage object at 0x000001FDD1F8A2E8>> = <Page.sys_user_manage.Sys_user_manage object at 0x000001FDD1F8A2E8>.get_User_table_value_by_key\n\nTestsuit\\System_Settings\\User_Manage\\Test_User_add_update_query.py:1071: AssertionError"}, "description": "\n                  测试用例编号:\n                  测试用例标题:通过状态查询已启用角色\n                  前置条件：打开编辑账号信息界面\n                  步骤：\n                  \"\n                  1.选择状态“已禁用”\n                  2.点击查询按钮\n          \n                  预期：\n                  1. 可以选择\n                  2. 返回模糊查询结果\n                  \n        ", "attachments": [{"source": "6be09dd6-ff47-4e1e-aa94-1158de374eb0-attachment.attach"}, {"source": "0afc52af-ea27-42de-89a8-193174e528e6-attachment.attach"}, {"source": "bb5904e3-d6f2-45d5-bdd6-f747933cd90e-attachment.attach"}, {"source": "beebe2ce-8ccc-4ab3-8672-220a64523793-attachment.attach"}, {"source": "edb4db6f-886f-467c-8b5f-6c2d52b6917d-attachment.attach"}, {"source": "3b461907-be58-43aa-b737-e23f39cf9ac7-attachment.attach"}, {"source": "0939a9ca-8111-468e-8611-64ac7a62a77e-attachment.attach"}, {"name": "失败的用例，已在异常处截图 :C:/Users/lyc/workspace/Web_Automation-master/Picture/1565339830.png 没有找到元素  by tag name,elements name is td", "source": "75847d54-7c84-461b-bd73-c1c673759d7b-attachment.png", "type": "image/png"}, {"source": "07a2254e-dbbe-4b0c-969c-5626a94222da-attachment.attach"}, {"source": "34474a51-190d-4c49-ba20-c6b7b2d43fe7-attachment.attach"}, {"source": "a9a35fce-445e-44b5-965b-17949d811e07-attachment.attach"}, {"source": "48d2b935-d624-4206-8017-f161b4187366-attachment.attach"}, {"source": "4c438777-6767-4c87-be22-9acc66524731-attachment.attach"}, {"source": "74a81fdd-9306-4f8e-aed3-c9113dc6f9c2-attachment.attach"}, {"source": "0fca65be-eee4-4175-8be7-f82dac92a4da-attachment.attach"}, {"source": "a2b0e3aa-06ee-42f5-85f5-e5d844958859-attachment.attach"}, {"source": "5b15c814-1cf6-4139-8891-e39760797c9a-attachment.attach"}, {"source": "92ec6545-ab76-4467-afa4-c1242eaabfe6-attachment.attach"}, {"source": "c4352ea8-d2ab-4868-b197-fa58003422d7-attachment.attach"}, {"source": "199af2b3-c84b-4cbf-938c-8b9e73b894a7-attachment.attach"}, {"name": "失败的用例，已在异常处截图 :C:/Users/lyc/workspace/Web_Automation-master/Picture/1565339839.png None", "source": "7384dae5-66bc-4d09-9eb7-be6503296bb0-attachment.png", "type": "image/png"}, {"name": "log", "source": "8f18d1c4-a06a-4c40-b16b-2d3fd7fcc64d-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "20e72ed0-e263-4b5c-8038-6878f1f4d47d-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "767028fb-33f8-42ed-a6a8-07a87bc8b0e8-attachment.txt", "type": "text/plain"}], "start": 1565339817043, "stop": 1565339839478, "uuid": "f76d091b-c241-4559-8b94-3d3cca9f04f3", "historyId": "50b3d5a4d1692bb316bb2f9215cf971e", "fullName": "Testsuit.System_Settings.User_Manage.Test_User_add_update_query.Test_user_add_update_query#test_query_by_status_No", "labels": [{"name": "story", "value": "通过状态查询已禁用角色"}, {"name": "feature", "value": "系统设置-用户管理-用户新增编辑查询"}, {"name": "parentSuite", "value": "Testsuit.System_Settings.User_Manage"}, {"name": "suite", "value": "Test_User_add_update_query"}, {"name": "subSuite", "value": "Test_user_add_update_query"}, {"name": "host", "value": "DESKTOP-DRJH5SS"}, {"name": "thread", "value": "11548-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testsuit.System_Settings.User_Manage.Test_User_add_update_query"}]}